### PR TITLE
Add release notes for Maven 3.8.3

### PR DESCRIPTION
**Pull request by michael-o**

### CODE REVIEW

**Comment by michael-o** [comment]
@rmannibucau Please check for the CDI API change.

File: content/markdown/docs/3.8.3/release-notes.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+If you have any questions, please consult:
+
+- the web site: [https://maven.apache.org/][2]
+- the maven-user mailing list: [https://maven.apache.org/mailing-lists.html](/mailing-lists.html)
+- the reference documentation: [https://maven.apache.org/ref/3.8.3/](/ref/3.8.3/)
+
+## Overview About the Changes
+
+This release mainly contains regression fixes from Maven 3.8.2 but it also [drops CDI API][6].
```
Comment (slawekjaranowski) [inline content/markdown/docs/3.8.3/release-notes.md:41]: You can also mention about improvement in maven speed like MNG-7235

Comment (michael-o) [reply content/markdown/docs/3.8.3/release-notes.md:41]: Makes sense, please propose the change.

Comment (slawekjaranowski) [reply content/markdown/docs/3.8.3/release-notes.md:41]: some think like:

> This release mainly contains regression fixes from Maven 3.8.2 and improvement in speed of parsing project [ref to MNG-7235]

File: pom.xml [hunk 2]
```diff
@@ -55,10 +55,10 @@
 
   <properties>
     <maven.site.path>maven-site</maven.site.path>
-    <currentStableVersion>3.8.2</currentStableVersion>
-    <currentStableReleaseDate>2021-08-04</currentStableReleaseDate>
-    <currentStableVersionDetails>ea98e05a04480131370aa0c110b8c54cf726c06f</currentStableVersionDetails>
-    <current38xVersion>3.8.2</current38xVersion>
+    <currentStableVersion>3.8.3</currentStableVersion>
+    <currentStableReleaseDate>2021-09-27</currentStableReleaseDate>
+    <currentStableVersionDetails>SHA-1</currentStableVersionDetails>
```
Comment (slawekjaranowski) [inline pom.xml:60]: it should be (probably)
```
ff8e977a158738155dc465c6a97ffaf31982d739 
```

**Pull request closed without merging**